spaceship/lib/spaceship/connect_api/models/device.rb

```diff
@@ -88,6 +88,50 @@ def self.create(client: nil, name: nil, platform: nil, udid: nil)
         resp = client.post_device(name: name, platform: platform, udid: udid)
         return resp.to_models.first
       end
+
+      # @param device_udid [String] Device Provisioning UDID that needs to be modified.
+      # @param client [ConnectAPI] ConnectAPI client.
+      # @param enabled [Boolean] New enabled value. true - if device must be enabled, `false` - to disable device. nil if no status change needed.
+      # @param new_name [String] A new name for the device. nil if no name change needed.
+      # @param platform [String] The platform of the device.
+      # @return (Device) Modified device based on the UDID of the device. nil if no device was found.
+      def self.modify(device_udid, client: nil, enabled: nil, new_name: nil, platform: nil)
+        client ||= Spaceship::ConnectAPI
+        existing = self.find_by_udid(device_udid, client: client, platform: platform, include_disabled: true)
+        return nil if existing.nil?
+
+        enabled = existing.enabled? if enabled.nil?
```

Comment:
No, it could not, cause it's if `enabled` is set, then `||=` will become `boolean or and assign` and will fail to disable the enabled device
```
var = false
var ||= true
puts var.to_s # true
```

Reply:
ohhh right 😄